slash/react/src/Alert/Alert.client.tsx

```diff
@@ -44,10 +45,16 @@ export const Alert = ({
   title,
   children,
   link,
+  iconSize = 24,
 }: PropsWithChildren<AlertProps>) => {
   return (
     <div className={`af-alert af-alert--${type}`}>
-      {getIconFromType(type)}
+      <Svg
+        src={getIconFromType(type)}
```

Comment:
Ce genre d'appels peuvent causer des appels en boucles.
Soit tu utilises un useMemo pour stocker la valeurs soit tu fais comme ceci : 
```suggestion
        src={() => getIconFromType(type)}
```